### ISSUE TITLE
Uninstallation guide

### ISSUE THREAD

**Issue by hoangtnm** [kubeflow/manifests#2299]
Hi, I tried to [install kubeflow with a single command](https://github.com/kubeflow/manifests#install-with-a-single-command) from README:

```bash
while ! kustomize build example | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
```
However, the docs doesn't clarify how to uninstall kubeflow after installation, or uninstall it completely if any error happens during the installation so that I can re-install again.

**Comment by viethung1234q** [comment]
Hi, I wonder if there is any update on how to uninstall Kubeflow ?

**Comment by juliusvonkohout** [comment]
I think this should be merged with #2216 

We are looking for your contribution :-)

/close

**Comment by bluecrayon52** [comment]
This worked for me: 
`while ! kustomize build example | kubectl delete -f -; do echo "Retrying to delete resources"; sleep 10; done`
